frontend/app/config/paths.js

```diff
@@ -76,6 +76,7 @@ module.exports = {
   appSrc: resolveApp("src"),
   appTsConfig: resolveApp("tsconfig.json"),
   appJsConfig: resolveApp("jsconfig.json"),
+  libSrc: resolveApp("../lib/src"),
```

Comment:
Get a reference to the lib's src folder. It's next to the app folder, so we move relatively down one folder to get to it.